free-1.md

```diff
@@ -10,34 +10,45 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I think the need being met here is and easy and user-friendly way to manipulate sounds and frequencies that the user records. As of now, I believe the only way that users can do what I'd like to have happen is by recording a sound, then using FL or other studio-type software to pitch bend to an appropriate frequency.  I don't believe there is an easy way to record and manipulate sounds in the way that I envision it.
+
+I imagine the experience as follows: A user records themselves singing or playing an instrument. They then use the DSL in order to add thirds, fourths, and other intervals as they choose. They can also do other simpler things such as slowing down and speeding up 
```

Comment:
Although I'm sure you get more into detail farther down, I'm currently wondering how you plan to interact with this DSL. The way you choose to interact with these features will determine how much alike or different this DSL is from other existing software.